docs/manual/en/introduction/Loading-3D-models.html

```diff
@@ -129,7 +129,7 @@ <h2>Troubleshooting</h2>
 			are available for
 			<a href="https://gltf-viewer.donmccurdy.com/" target="_blank" rel="noopener">three.js</a> and
 			<a href="https://sandbox.babylonjs.com/" target="_blank" rel="noopener">babylon.js</a>. If the model
-			appears correctly in one or more applications,
+			appears incorrectly in one or more applications,
```

Comment:
The premise is the model is not loaded correctly in `three.js`. If now the model can be displayed _correctly_ one or more other applications, there might be a bug in `three.js`. 

If none of the applications render the model correctly, the tool that exported the glTF asset is probably buggy.

So it seems the existing description is correct.

Reply:
That makes complete sense. Thank you for the clarification. Closing this PR as it's not needed.